src/main/java/com/thealgorithms/scheduling/FCFSScheduling.java

```diff
@@ -0,0 +1,47 @@
+package com.thealgorithms.scheduling;
+
+import com.thealgorithms.scheduling.entities.ProcessDetails;
+
+import java.util.List;
+
+/**
+ * Non-pre-emptive First Come First Serve scheduling
```

Comment:
Could you add a link to wikipedia or any other source where the algorithm is explained?

Reply:
Yes, have added a source here where everything would be explained.